### PR TITLE
added bot to reviewers list

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
to speed up operatorhub packages publishing adding victoriametrics-bot to reviewers list

### CODE REVIEW

File: config/manifests/ci.yaml [hunk 1]
```diff
@@ -7,3 +7,4 @@ reviewers:
   - f41gh7
   - AndrewChubatiuk
   - Haleygo
+  - victoriametrics-bot
```
Comment (f41gh7) [inline config/manifests/ci.yaml:10]: As far as I remember it also must be manually edited at operator-hub with a separate PR. 

This change shouldn't be included at commit with CRD changes.

Comment (AndrewChubatiuk) [reply config/manifests/ci.yaml:10]: I've added user manually, in operatorhub repo, but actually it's managed with our release pipeline as well
`make olm` puts ci.yaml into release archive
https://github.com/VictoriaMetrics/operator/blob/593d5237127af436087245a036970145ef4042f4/Makefile#L221

**Review by f41gh7 — APPROVED** [review]
LGTM

**Comment by f41gh7** [comment]
Thanks!

**Pull request merged**